.travis.yml

```diff
@@ -3,13 +3,14 @@ sudo: false # We want container builds.
 language: python
 python:
   - "2.7"
-node_js:
-  - "0.10"
+env:
+  - NODEJS_VERSION="0.10"
+  - NODEJS_VERSION="4.5.0"
 matrix:
   include:
     # Test on pypy without coverage, because it's unnecessary and very slow.
     - python: "pypy"
-      env: PYPY_VERSION="5.3.1" NO_COVERAGE="1"
+      env: PYPY_VERSION="5.3.1" NO_COVERAGE="1" NODEJS_VERSION="4.5.0"
```

Comment:
Should we maybe try things working with latest stable node (6.4.0) while we're at it?